File: Euler5/Euler5.java
public class Euler5 {
    public static void main(String[] args){
     System.out.println(findDividen());  
    }
    private static int findDividen(){
        int i = 20;
        while(true){
           if (i % 19 == 0 && i % 18 == 0 && i % 17 == 0 && i % 16 == 0 && i % 14 == 0 && i % 13 == 0 && i % 11 == 0) return i;
           i += 20;
        }
    }
}

File: Python/euler5.py
def euler5():
    focus = 1
    works = False
    while not works:
        for x in range(2, 21):
            works = True
            if (focus % x) != 0:
                works = False
                focus *= lcm(focus, x)
                break 
    print(focus)

def lcm(n, k):
    done = False
    num = n
    i = 1
    while not done:
        if (num*i) % k != 0:
            i += 1
            continue
        return i

euler5()